test/k8sT/Services.go

```diff
@@ -688,6 +757,77 @@ var _ = Describe("K8sServicesTest", func() {
 			doRequestsExpectingHTTPCode(url, count, "503", k8s2Name)
 		}
 
+		testIPv4FragmentSupport := func() {
+			var data v1.Service
+			k8s1Name, k8s1IP := getNodeInfo(helpers.K8s1)
+			k8s2Name, k8s2IP := getNodeInfo(helpers.K8s2)
+
+			waitPodsDs()
+
+			getPodOnNode := func(nodeName string, podFilter string) (string, string) {
+				var podName string
+				podsNodes, err := kubectl.GetPodsNodes(helpers.DefaultNamespace, fmt.Sprintf("-l %s", podFilter))
+				ExpectWithOffset(2, err).Should(BeNil(), "Cannot retrieve pods nodes with filter %q", podFilter)
+				Expect(podsNodes).ShouldNot(BeEmpty(), "No pod found in namespace %s with filter %q", helpers.DefaultNamespace, podFilter)
+				for pod, node := range podsNodes {
+					if node == nodeName {
+						podName = pod
+						break
+					}
+				}
+				ExpectWithOffset(2, podName).ShouldNot(BeEmpty(), "Cannot retrieve pod on node %s with filter %q", nodeName, podFilter)
+				podsIPs, err := kubectl.GetPodsIPs(helpers.DefaultNamespace, podFilter)
+				ExpectWithOffset(2, err).Should(BeNil(), "Cannot retrieve pods IPs with filter %q", podFilter)
+				Expect(podsIPs).ShouldNot(BeEmpty(), "No pod IP found in namespace %s with filter %q", helpers.DefaultNamespace, podFilter)
+				podIP := podsIPs[podName]
+				return podName, podIP
+			}
+
+			// Get testDSClient and testDS pods running on k8s1.
+			// This is because we search for new packets in the
+			// conntrack table for node k8s1.
+			clientPod, clientIP := getPodOnNode(helpers.K8s1, testDSClient)
+			_, serverIP := getPodOnNode(helpers.K8s1, testDS)
+
+			err := kubectl.Get(helpers.DefaultNamespace, "service test-nodeport").Unmarshal(&data)
+			Expect(err).Should(BeNil(), "Cannot retrieve service")
+			nodePort := data.Spec.Ports[1].NodePort
+
+			// TODO: We want TargetPort but it is of type
+			// intstr.IntOrString, so the following does not work.
+			// Not sure how to unwrap it.
+			//serverPort := data.Spec.Ports[1].TargetPort
```

Comment:
Seems like this should just require `IntValue()`?

https://godoc.org/k8s.io/apimachinery/pkg/util/intstr#IntOrString.IntValue

Reply:
Oh, looks like the function I was looking for. Thank you for that, and for the review!